.gitignore

```diff
@@ -244,5 +244,5 @@ fabric.properties
 *.drawio.dtmp
 
 # Secureli-generated files (do not modify):
-.secureli
+.secureli/local
```

Comment:
You will need `.secureli/logs` here as well.

Reply:
I moved the logs folder into `local` (for new installations anyway). But I think I'm actually going to revert some of this work and keep the original locations to simplify the migration/reduce breaking changes.